core/src/test/java/io/grpc/internal/ChannelTracerTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2018 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.internal;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import io.grpc.internal.Channelz.ChannelStats;
+import io.grpc.internal.Channelz.ChannelTrace.Event;
+import io.grpc.internal.Channelz.ChannelTrace.Event.Severity;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Unit tests for {@link ChannelTracer}.
+ */
+@RunWith(JUnit4.class)
+public class ChannelTracerTest {
+  @Rule
+  public final ExpectedException thrown = ExpectedException.none();
+
+  @Test
+  public void channelTracerWithZeroMaxEventsShouldThrow() {
+    thrown.expect(IllegalArgumentException.class);
+    thrown.expectMessage("maxEvents must be greater than zero");
+
+    new ChannelTracer(0 /* maxEvents*/, 3L /* channelCreationTimeNanos */);
```

Comment:
nit: use `/*argName=*/` because the linter will check when the label is correct.

Reply:
Done.